src/main/java/org/folio/rs/domain/entity/RetrievalQueueRecord.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.rs.domain.entity;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Table(name = "retrieval_queue")
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class RetrievalQueueRecord {
```

Comment:
Please, double check that all needed fields are presented: `<?xml~version="1.0"encoding="UTF-8"~standalone="yes"?><asrRequests><asrRequests><asrRequest><holdId>354167</holdId><itemBarcode>~82703197</itemBarcode><title>~Philologia antiqua.</title><author></author><callNumber>~PA1.A1P45</callNumber><patronBarcode>~7137806</patronBarcode><patronName>~Bottorff,David</patronName><requestDate>~2020-07-29 00:00:00.0</requestDate><pickupLocation>~JRLMAIN</pickupLocation><requestStatus>~1</requestStatus><requestNote></requestNote></asrRequest></asrRequests></asrRequests>~`